runner/runners/invoke.go

```diff
@@ -334,6 +339,11 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 		stdlog.Write([]byte(header))
 	}
 
+	if inv.diskMonitor != nil {
+		inv.diskMonitor.GetStartSizes()
```

Comment:
because this is after the snapshot checkout, we aren't tracking the disk usage increase associated with it. is that intended?

Reply:
yes - this is just for the command execution